model/evergreen_config.go

```diff
@@ -0,0 +1,28 @@
+package model
```

Comment:
We can get rid of this, it is a duplicate of https://github.com/evergreen-ci/cedar/blob/main/model/config.go#L65-L73.

Reply:
I moved it to the evergreen package because we still need an options struct.